sei_for_cosmovisor

```diff
@@ -0,0 +1,537 @@
+echo DAEMON_HOME:"$DAEMON_HOME"
+echo SEIDVER:"$SEIDVER"
+echo VISORVER:"$VISORVER"
+echo '================================================='
+read -r -s -n 1 -p "Are the above values correct? If true, press a key. If false, exit with ctrl+c"
+
+sudo systemctl stop seid
+sudo systemctl disable seid
+sudo systemctl disable seid.service
+sudo systemctl daemon-reload
+echo -e "\e[1m\e[32m old seid services disable and stop\e[0m" && sleep 1
+
+sleep 1
+echo -e "\e[1m\e[32m1. Update and Upgrade check... \e[0m" && sleep 1
+# update
+sudo apt update && sudo apt upgrade -y
+
+echo -e "\e[1m\e[32m2. installing tools and libs... \e[0m" && sleep 1
+# packages
+sudo apt install curl tar wget clang pkg-config libssl-dev jq build-essential bsdmainutils git make ncdu gcc git jq chrony tmux net-tools liblz4-tool -y
+
+sleep 1
+cd $HOME || { echo "$HOME dizinine girilemiyor"; sleep 10; exit 13;}
+wget -O go1.18.1.linux-amd64.tar.gz https://golang.org/dl/go1.18.1.linux-amd64.tar.gz
+rm -rf /usr/local/go && tar -C /usr/local -xzf go1.18.1.linux-amd64.tar.gz && rm go1.18.1.linux-amd64.tar.gz
+echo 'export GOROOT=/usr/local/go' >> "$HOME"/.bash_profile
+echo 'export GOPATH=$HOME/go' >> "$HOME"/.bash_profile
+echo 'export GO111MODULE=on' >> "$HOME"/.bash_profile
+echo 'export PATH=$PATH:/usr/local/go/bin:$HOME/go/bin' >> "$HOME"/.bash_profile && . "$HOME"/.bash_profile
+go version
+source "$HOME"/.bash_profile
+
+echo -e "\e[1m\e[32m3. sei-chain library download and install... \e[0m"
+
+# download binary
+cd $HOME || { echo "$HOME dizinine girilemiyor"; sleep 10; exit 13;}
+echo git clone --depth 1 --branch "$SEIDVER" https://github.com/sei-protocol/sei-chain.git
+git clone --depth 1 --branch "$SEIDVER" https://github.com/sei-protocol/sei-chain.git
+cd sei-chain && make install
+go build -o build/seid ./cmd/seid
+
+#unnecessary maybe necessary
+#chmod +x ./build/seid && sudo cp ./build/seid /usr/local/bin/seid
+
+sleep 1
+
+#impornant
+cp "$HOME"/go/bin/seid /usr/local/bin/
+
+mv "$HOME"/.sei-chain "$HOME"/.sei
+
+#if not exist /usr/local/bin/seid write ERROR
+#if exist /usr/local/bin/seid and version NOT not match write ERROR
+if [ ! -f "/usr/local/bin/seid" ]; then
+    echo "/usr/local/bin/seid FILE NOT EXIST"
+    read -r -s -n 1 -p "Press any key to EXIT . . ."
+    exit 13
+else
+  SEIDBUILDVER=$(/usr/local/bin/seid version)
+  if [ "$SEIDBUILDVER" == "$SEIDVER" ]; then
+  	echo /usr/local/bin/seid version "$SEIDVER"
+  else
+    	echo -e "\e[1m\e[31m2. Error version not match $SEIDVER \e[0m"
+    	echo -e "\e[1m\e[31m2. please check /usr/local/bin/seid version file \e[0m"
+    	read -r -s -n 1 -p "Press any key to EXIT . . ."
+  	exit 13
+  fi
+fi
+
+# config
+seid config chain-id "$CHAIN_ID"
+seid config keyring-backend file
+
+# init
+seid init "$NODENAME" --chain-id "$CHAIN_ID"
+
+# download genesis and addrbook
+wget -qO "$HOME"/.sei/config/genesis.json "https://raw.githubusercontent.com/sei-protocol/testnet/main/sei-testnet-2/genesis.json"
```

Comment:
we may want to add a CHAIN_ID variable so we can reuse this script.
E.g. 
CHAIN_ID=sei-testnet-2
`"https://raw.githubusercontent.com/sei-protocol/testnet/main/$CHAIN_ID/genesis.json"`